### DISCUSSION TITLE
[feat req]: Merging MotherDuck databases other than the default db into postgres public schema

### DISCUSSION THREAD

**Discussion by naoyak** [duckdb/pg_duckdb#459] [Ideas]
When using `pg_duckdb` with MotherDuck, we'd like to select a db other than the internally configured default (`my_db`) to merge into the `postgres.public` schema.

Current behavior:

```markdown
1. Each schema in your default MotherDuck database are simply merged with the Postgres schemas with the same name.
2. Except for the `main` DuckDB schema in your default database, which is merged with the Postgres `public` schema.
3. Tables in other databases are put into dedicated DuckDB-only schemas. These schemas are of the form `ddb$<duckdb_db_name>$<duckdb_schema_name>` (including the literal `$` characters).
4. Except for the `main` schema in those other databases. That schema should be accessed using the shorter name `ddb$<db_name>` instead.
```

Since there isn't a user-facing public API for a MotherDuck user to reassign the "default" database, we're stuck with the verbose `ddb$other_md_db$schema` naming scheme if we want to consume other schemas/tables downstream. Could we add a configuration setting e.g. in `postgresql.conf` to manually select which MotherDuck db to use as the "default"?

**Comment by JelteF** [comment]
Thanks for the report, we realized this was a problem as well and are planning to add an option for this soon to the `postgresql.conf` file.

**Comment by naoyak** [comment]
Thanks @JelteF, opened #470.

**Comment by naoyak** [comment]
Added in #470
